drivers/SmartThings/harman-luxury/config.yaml

```diff
@@ -0,0 +1,5 @@
+name: "Harman-Luxury"
```

Comment:
This key should be a "human friendly" name, and as such doesn't need the hyphen.